From c1e9d092f44da5411e3cf1b3e004ecc9a7daa010 Mon Sep 17 00:00:00 2001
From: Omri Itach <omrii@marvell.com>
Date: Thu, 30 Jun 2016 14:05:43 +0300
Subject: [PATCH 0576/2241] usb: a80x0: enable USB3.0 Host 0 on CP1

CP1 support only single USB port.
(USB Host 1 is not connected to CP1)

Change-Id: Id782fdf60988ead588aae187e86802e1b872955b
Signed-off-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30864
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-db-router.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
index 3165369..166a155 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
@@ -114,6 +114,9 @@
 	};
 	cpn-110-slave {
 		config-space {
+			usb3h0_1: usb3@500000 {
+				status = "okay";
+			};
 			sata@540000 {
 				status = "okay";
 			};
-- 
2.7.4

